configure.py

```diff
@@ -1827,7 +1827,7 @@ def configure_abseil(build_dir, mode):
         build $builddir/{mode}/dist/tar/scylla-tools-package.tar.gz: copy tools/java/build/scylla-tools-package.tar.gz
         build $builddir/{mode}/dist/tar/scylla-jmx-package.tar.gz: copy tools/jmx/build/scylla-jmx-package.tar.gz
 
-        build dist-{mode}: phony dist-server-{mode} dist-python3 dist-tools dist-jmx
+        build dist-{mode}: phony dist-server-{mode} dist-python3 dist-tools dist-jmx dist-unified
```

Comment:
This is unfortunately not sufficient. The `dist-<mode>` target is supposed to build artifacts just for that target, but `dist-unified` will build the artifacts for _all_ configured build modes. So this change should be `dist-unified-{mode}`, but that build target does not exist (yet) and needs to be added.